Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -734,73 +752,77 @@ theorem ofFunction_union_of_top_of_nonempty_inter {s t : Set α}
     refine' ⟨ht, ENNReal.tsum_le_tsum fun n => _⟩
-    cases h
-    exacts[h (image_preimage_subset _ _), (congr_arg m (h.image_preimage (t n))).le]
+    exact h.elim
+      (· (image_preimage_subset _ _))
+      fun h => (congr_arg m (h.image_preimage (t n))).le
 #align measure_theory.outer_measure.comap_of_function MeasureTheory.OuterMeasure.comap_ofFunction
 
 theorem map_ofFunction_le {β} (f : α → β) :
     map f (OuterMeasure.ofFunction m m_empty) ≤
       OuterMeasure.ofFunction (fun s => m (f ⁻¹' s)) m_empty :=
   le_ofFunction.2 fun s => by
     rw [map_apply]
-    apply of_function_le
+    apply ofFunction_le
 #align measure_theory.outer_measure.map_of_function_le MeasureTheory.OuterMeasure.map_ofFunction_le
 
 theorem map_ofFunction {β} {f : α → β} (hf : Injective f) :
     map f (OuterMeasure.ofFunction m m_empty) =
       OuterMeasure.ofFunction (fun s => m (f ⁻¹' s)) m_empty := by
-  refine' (map_of_function_le _).antisymm fun s => _
-  simp only [of_function_apply, map_apply, le_infᵢ_iff]
+  refine' (map_ofFunction_le _).antisymm fun s => _
+  simp only [ofFunction_apply, map_apply, le_infᵢ_iff]
   intro t ht
   refine' infᵢ_le_of_le (fun n => range fᶜ ∪ f '' t n) (infᵢ_le_of_le _ _)
-  · rw [← union_Union, ← inter_subset, ← image_preimage_eq_inter_range, ← image_Union]
+  · rw [← union_unionᵢ, ← inter_subset, ← image_preimage_eq_inter_range, ← image_unionᵢ]
     exact image_subset _ ht
   · refine' ENNReal.tsum_le_tsum fun n => le_of_eq _
     simp [hf.preimage_image]
 #align measure_theory.outer_measure.map_of_function MeasureTheory.OuterMeasure.map_ofFunction
 
 theorem restrict_ofFunction (s : Set α) (hm : Monotone m) :
     restrict s (OuterMeasure.ofFunction m m_empty) =
-      OuterMeasure.ofFunction (fun t => m (t ∩ s)) (by rwa [Set.empty_inter]) := by
-  simp only [restrict, LinearMap.comp_apply, comap_of_function _ (Or.inl hm),
-    map_of_function Subtype.coe_injective, Subtype.image_preimage_coe]
+      OuterMeasure.ofFunction (fun t => m (t ∩ s)) (by simp only [empty_inter, m_empty]) := by
+      rw [restrict]
+      simp only [LinearMap.comp_apply]
+      rw [comap_ofFunction _ (Or.inl hm)]
+      simp only [map_ofFunction Subtype.coe_injective, Subtype.image_preimage_coe]
 #align measure_theory.outer_measure.restrict_of_function MeasureTheory.OuterMeasure.restrict_ofFunction
 
 theorem smul_ofFunction {c : ℝ≥0∞} (hc : c ≠ ∞) :
     c • OuterMeasure.ofFunction m m_empty = OuterMeasure.ofFunction (c • m) (by simp [m_empty]) :=
   by
   ext1 s
-  haveI : Nonempty { t : ℕ → Set α // s ⊆ ⋃ i, t i } := ⟨⟨fun _ => s, subset_Union (fun _ => s) 0⟩⟩
-  simp only [smul_apply, of_function_apply, ENNReal.tsum_mul_left, Pi.smul_apply, smul_eq_mul,
-    infᵢ_subtype', ENNReal.infᵢ_mul_left fun h => (hc h).elim]
+  haveI : Nonempty { t : ℕ → Set α // s ⊆ ⋃ i, t i } := ⟨⟨fun _ => s, subset_unionᵢ (fun _ => s) 0⟩⟩
+  simp only [smul_apply, ofFunction_apply, ENNReal.tsum_mul_left, Pi.smul_apply, smul_eq_mul,
+  infᵢ_subtype']
```

Comment:
Indentation

Reply:
```
theorem smul_ofFunction {c : ℝ≥0∞} (hc : c ≠ ∞) :
    c • OuterMeasure.ofFunction m m_empty = OuterMeasure.ofFunction (c • m) (by simp [m_empty]) :=
  by
  ext1 s
  haveI : Nonempty { t : ℕ → Set α // s ⊆ ⋃ i, t i } := ⟨⟨fun _ => s, subset_unionᵢ (fun _ => s) 0⟩⟩
  simp only [smul_apply, ofFunction_apply, ENNReal.tsum_mul_left, Pi.smul_apply, smul_eq_mul,
  infᵢ_subtype']
  rw [ENNReal.infᵢ_mul_left fun h => (hc h).elim]
```

`by` has its own line because the line before has spans 99 characters.

What should I do here?

Sorry, I'm a new contributor. Please bear with me. Where can I find the documentation about indentation rules?

Reply:
If `simp [...]` spills over onto another line, because you have many lemmas, the subsequent lines should have an additional indentation.